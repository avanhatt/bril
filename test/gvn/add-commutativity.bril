int main(x : int, y : int) {
    z : int = add x y;
    z2 : int = add x y;
    z3 : int = add y x;
    print z2;
    print z3;
    ret z;
}