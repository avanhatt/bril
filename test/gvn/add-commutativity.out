int main(x : int, y : int) {
b1:
  z1: int = add x y;
  print z1;
  print z1;
  ret z1;
}
